src/Components/BottomButtons.jsx

```diff
@@ -0,0 +1,12 @@
+const BottomButtons = () => { 
+    return ( 
+        <div className="BottomButtonPane"> 
+            <button> Home </button>
```

Comment:
Can we allow a user to navigate back to home from a single article here? check out the react router notes again (have a look at links especially)